modules/ingestor/src/service/advisory/osv/loader.rs

```diff
@@ -22,10 +22,12 @@ impl<'g> OsvLoader<'g> {
         &self,
         location: L,
         record: R,
-    ) -> Result<(), Error> {
+    ) -> Result<String, Error> {
```

Comment:
Nice. I always feel like `Result<(), ...>` is wrong, or at least benefits from more thought.